src/screens/Technology/components/Details/index.tsx

```diff
@@ -121,6 +128,119 @@ export const Characteristics = () => {
   );
 };
 
+interface MarkerProps {
+  type: string
+  [key: string]: any
+}
+
+export const Geo = (): JSX.Element => {
+  const technology = useTechnology();
+  const types: string[] = ['who_develop', 'where_can_be_applied', 'where_is_already_implemented'];
+  const region = {
+    latitude: -6.780127,
```

Comment:
It should be based on technology (or user, I don't know) location, right?

Reply:
@mateus4k puxei esse valor da web, mas o @nichollasrennah comentou sobre centralizar na região do semiárido